eisbuk-admin/src/store/selectors/calendar.ts

```diff
@@ -0,0 +1,49 @@
+import { LocalStore } from "@/types/store";
+
+interface calendarSlot {
```

Comment:
Types should be PascalCase, (i.e. `CalendarSlot`)...this is a convention, but it's also there to distinguish the often used same naming of function and function interface (the example below is a very simplified usage, and, in that case, I would declare the function interface inline, but this format is used for more complex functions to visually separate type interface and function implementation for better readability)
```typescript
interface GetTwo {
  (): number
}

const getTwo: GetTwo () => 2
```